### ISSUE TITLE
Loudness normalization (automatic volume adjustment)

### ISSUE THREAD

**Issue by Sogolumbo** [bydariogamer/beatgame#35]
We should use [Loudness normalization](https://en.wikipedia.org/wiki/Audio_normalization#Loudness_normalization) to make sure all songs are played at an equal loudness.
I'd suggest roughly `−14 LUFS` ([comparison](https://en.wikipedia.org/wiki/Loudness_war#2020s)).
(The player can still turn down the volume after that).

**Comment by bydariogamer** [comment]
Good idea! We can just use the average of the matrix to get the number and then set the volume... There is a little inconvenient however... the song is played at 1.0, so a song with low volume would have to stay low...
Unless... We can always set the volume so for a song in the right volume the `-14 LUFS` is the 0.7 of the volume in `pygame` (in example) or something like that.
